app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -43,14 +43,22 @@ def options
   end
 
   def should_display_account_reset_or_cancel_link?
-    # IAL2 users should not be able to reset account to comply with AAL2 reqs
-    !current_user.decorate.identity_verified?
+    # IAL2 non-docauth users should not be able to reset account to comply with AAL2 reqs
```

Comment:
This conditional is a little tough to follow; could we break it down somewhat to make it easier to read?